pod/perldelta.pod

```diff
@@ -64,6 +64,43 @@ as with C<m?pattern?>.
 
 [GH #20763]
 
+=head2 Calling the import method of an unknown package produces an error
+
+Historically it has been possible to call the import() or unimport()
+method of any class, including ones which have not been defined, with
+an argument and not experience an error. For instance this code will
+not throw an error in 5.38:
+
+    Class::That::Does::Not::Exist->import("foo");
+
+however, as of 5.39.1 this will throw an exception.  Note that calling
+these methods with no arguments continues to silently succeed and do
+nothing. For instance
+
+    Class::That::Does::Not::Exist->import();
+
+will continue to not throw an error. This is because every class
+implicitly inherits from the class UNIVERSAL which now defines an import
+method. In older perls there was no such method defined, and instead the
+method calls for C<import> and C<unimport> were special cased to not
+throw errors if there was no such method defined.
+
+This change has been added because it makes it easier to detect case
+typos in C<require> and C<use> statements when running on
```

Comment:
Why mention `require` here? AFAIK it does not interact with `import`.

Reply:
Ah, good catch.